### DISCUSSION TITLE
This guy Copy my game

### DISCUSSION THREAD

**Discussion by zhengbilly** [nesbox/TIC-80#1527] [Q&A]
Hello,I'm QKboy. This guy "WONDERCONTER32"copy my game just change the auther name.Please delete it. Its address is https://tic80.com/play?cart=1990    My game's address is https://tic80.com/play?cart=1618

**Comment by nesbox** [comment]
Hi, 
I see and will delete it in 1-2 hrs
Thank you.

**Comment by nesbox** [comment]
Done!
